app/views/play/cinematic/CinematicLankBoss.js

```diff
@@ -17,164 +37,289 @@ const Lank = require('lib/surface/Lank')
  * @implements {System}
  */
 export default class CinematicLankBoss {
```

Comment:
In general I am not entirely happy with this file. There is re-used and hard coded code paths. Especially with attaching lanks. I feel there is better, reusable abstractions for moving lanks and playing actions on lanks. However I didn't want to over-abstract when we don't know how this system will be received and what changes will be required.
It's not easy to treat the characters/background, and thang background object as the same thing (although they are all Lanks). This is due to all the different feature requirements. Characters  have Actions. Backgrounds need scaling. Background objects pop in and out after delays.

The system follows the high level design outlined [here](https://codecombat.slab.com/posts/cinematic-system-design-2cg8x7si#system-a-high-level).